src/flick/lst/models.py

```diff
@@ -3,16 +3,18 @@
 from django.db import models
 
 from show.models import Show
+from user.models import Profile
 
 
 class Lst(models.Model):
     lst_name = models.CharField(max_length=100)
-    lst_pic = models.ForeignKey(AssetBundle, on_delete=models.CASCADE, blank=True, null=True)
-    is_favorite = models.BooleanField(default=False)
-    is_private = models.BooleanField(default=False)
-    is_watched = models.BooleanField(default=False)
-    collaborators = models.ManyToManyField(User, related_name="collaborators", blank=True)
-    owner = models.ForeignKey(User, related_name="owner", on_delete=models.CASCADE)
+    lst_pic = models.TextField(blank=True, null=True)
+    # lst_asset_bundle = models.ForeignKey(AssetBundle, on_delete=models.CASCADE, blank=True, null=True)
+    is_favorite = models.BooleanField(default=False, null=True)
+    is_private = models.BooleanField(default=False, null=True)
+    is_watched = models.BooleanField(default=False, null=True)
```

Comment:
not sure how a list can be "is watched"? maybe an individual movie can be is watched, or how about having a timestamp of "last watched"? that can enable some analysis later like "you like watching these lists at night; these are your favorite lists last week"... just a thought

Reply:
I see that maybe we are making "is watched" a check-box thing that users can change, but I still think maybe we can keep a time stamp

Reply:
really like the analysis idea -- but i think this was just gonna be a list full of shows you've watched if that makes sense.. but also how are we going to deal with watched shows individually and does this actually matter? like do ppl want to mark shows as watched or would they rather add to lists that are "watched" ?

Reply:
I'll put in an issue to add a timestamp for is_watched, which gets updated whenever the user changes it to True